src/featuretransforms.jl

```diff
@@ -0,0 +1,154 @@
+FeatureTransforms.is_transformable(::KeyedDataset) = true
```

Comment:
This is now automatic when an `apply` method is defined for a type

https://github.com/invenia/FeatureTransforms.jl/releases/tag/v0.3.4